UnityPy/export/Texture2DConverter.py

```diff
@@ -356,7 +363,7 @@ def rg(
     padding = bytes(padding_size)
     rgb_data = b"".join(
         stream.read(padding_size * 2) + padding
-        for _ in range(image_data / (2 * padding_size))
+        for _ in range(len(image_data) // (2 * padding_size))
```

Comment:
😓 ooph, thanks for catching that one, looks like I accidentally deleted my commit with the corrected version.